crates/reth/db/src/rocksdb/schema.rs

```diff
@@ -8,3 +8,13 @@ define_table_with_seek_key_codec!(
     /// store of block witness data. Data stored as serialized bytes for directly serving in rpc
     (BlockWitnessSchema) B256 => Vec<u8>
 );
+
+define_table_with_seek_key_codec!(
+    /// store of block state diff data. Data stored as serialized bytes for directly serving in rpc
+    (BlockStateDiffSchema) B256 => Vec<u8>
+);
+
+define_table_with_seek_key_codec!(
+    /// block number => hash mapping for easier testing.
+    (BlockNumberByHash) u64 => Vec<u8>
```

Comment:
Shouldn't this be `u64 => B256`?  And wouldn't it be called `BlockHashByNumber`?  I'm not sure of the real schema here.

Reply:
I'm not sure why, but having `B256` as a value (instead of `Vec<u8>`) doesn't work with macro.

Reply:
Oh is B256 not serializable according to their codec types?

Reply:
Well, it seems like it's not serializable.